src/app/language/en.json

```diff
@@ -1,4 +1,13 @@
 {
+	"1": "1",
+	"2": "2",
+	"3": "3",
+	"4": "4",
+	"5": "5",
+	"6": "6",
+	"7": "7",
+	"8": "8",
+	"9": "9",
```

Comment:
Can we remove these 1-9's. Its not something that should go to transifex for translation.

Reply:
sure, they were added automatically, so I thought it belongs to the PR, will remove